benchmark/tool/C/FindPaddle.cmake

```diff
@@ -0,0 +1,18 @@
+set(PADDLE_ROOT $ENV{PADDLE_ROOT} CACHE PATH "Paddle Path")
+find_path(PADDLE_INC NAMES capi.h PATHS ${PADDLE_ROOT}/include/paddle)
+find_library(PADDLE_LIB NAMES paddle_capi_shared PATHS ${PADDLE_ROOT}/lib/${ANDROID_ABI})
+if(PADDLE_INC AND PADDLE_LIB)
+  message("Found")
+else()
+  message(${PADDLE_ROOT})
+  message("Not Found")
```

Comment:
Suggest to use `message(STATUS ...)`, and the message should be more detail, such as 
```
message(STATUS "Found PaddlePaddle (include: ${PADDLE_INC}; library: ${PADDLE_LIB})")
message(FATAL_ERROR "Cannot find PaddlePaddle on ${PADDLE_ROOT}")
```

Reply:
Done.